### DISCUSSION TITLE
My custom made PNG carts

### DISCUSSION THREAD

**Discussion by Weeppiko** [nesbox/TIC-80#2549] [Show and tell]
I thought about sharing my carts in the png format, but once looking at the exported cart I didn't really felt it. I mean, there is not wrong with it, it's just that it doesn't fit me personally, so I went to the possibility of replacing them with something else.

After discussing about it on discord, some researching about the png format, and the fact that steganography is not used anymore, the results are these silly tiny carts based on Famicom cartridges:

![famigato](https://github.com/nesbox/TIC-80/assets/88540876/ce82da3c-f2e4-4985-9652-4fc16b799e3a) ![hangon](https://github.com/nesbox/TIC-80/assets/88540876/6610ca22-f3f1-43d5-9630-3de701800aee) ![sonicmini](https://github.com/nesbox/TIC-80/assets/88540876/ea6ce706-a6b5-47d4-9393-429ad3436e62)

I made them by simply adding the `caRt` chunk on the existing png file, which means it should work with any arbitrary image, and TIC-80 will recognize them without any issues. Be aware that saving the cart will replace it to the default image.

**Comment by bryce-carson** [comment]
Very, very cool.
